test/channels/api.js

```diff
@@ -102,30 +101,41 @@ describe('api', function() {
       ;
     });
 
+    it('returns a custom origin CORS header', function(done) {
+      process.env.CORS_ENABLED = 'true';
+      process.env.CORS_ORIGIN = "localhost";
+      api = uut([recipe], connections, codecs, channels)();
+      request = supertest(api.app);
+
+      request
+      .get('/file')
+      .expect('Access-Control-Allow-Origin', 'localhost')
+      .end(done)
+      ;
+    });
+
     it('returns an origin-mirroring CORS header', function(done) {
-      process.env.CORS_ENABLED = true;
-      process.env.CORS_ORIGIN = 'http://localhost';
+      process.env.CORS_ENABLED = 'true';
+      process.env.CORS_ORIGIN = 'true';
       api = uut([recipe], connections, codecs, channels)();
       request = supertest(api.app);
 
       request
       .get('/file')
       .set('Origin', 'localhost')
-      .expect('Access-Control-Allow-Origin', 'http://localhost')
+      .expect('Access-Control-Allow-Origin', 'localhost')
       .end(done)
```

Comment:
Might be good to chain in another test that uses another origin, just to be on the safe side. Something like:

```javascript
request
.get('/file')
.set('Origin', 'localhost')
.expect('Access-Control-Allow-Origin', 'localhost')
.end(() => {
  request
  .get('/file')
  .set('Origin', 'localhost2')
  .expect('Access-Control-Allow-Origin', 'localhost2')
  .end(done)
  ;
})
;
```